app/src/main/res/drawable/menu_icons_camera.xml

```diff
@@ -0,0 +1,16 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="25dp"
+    android:height="25dp"
+    android:viewportWidth="25"
+    android:viewportHeight="25">
+  <group>
+    <clip-path
+        android:pathData="M0,0h25v25H0z"/>
+    <path
+        android:pathData="M24.17,5.73a2.69,2.69 0,0 0,-1.97 -0.83h-3.95v-0.05a2.17,2.17 0,0 0,-2.19 -2.19L8.94,2.66a2.19,2.19 0,0 0,-2.21 2.19v0.05L2.8,4.9c-0.78,0 -1.47,0.3 -1.97,0.83 -0.5,0.5 -0.83,1.2 -0.83,1.97v11.83c0,0.79 0.3,1.48 0.83,1.98 0.5,0.5 1.21,0.83 1.97,0.83h19.4c0.78,0 1.47,-0.31 1.97,-0.83 0.5,-0.5 0.83,-1.22 0.83,-1.98L25,7.7c0,-0.78 -0.3,-1.47 -0.83,-1.97zM23.77,19.53h-0.03c0,0.43 -0.17,0.81 -0.45,1.1 -0.29,0.28 -0.67,0.45 -1.1,0.45L2.8,21.08c-0.42,0 -0.8,-0.17 -1.09,-0.45a1.53,1.53 0,0 1,-0.45 -1.1L1.26,7.7c0,-0.43 0.17,-0.8 0.45,-1.1 0.29,-0.28 0.67,-0.45 1.1,-0.45h4.58c0.36,0 0.64,-0.28 0.64,-0.64v-0.69c0,-0.26 0.1,-0.5 0.26,-0.66 0.17,-0.17 0.4,-0.26 0.67,-0.26h7.1c0.27,0 0.5,0.1 0.67,0.26 0.17,0.17 0.26,0.4 0.26,0.66v0.7c0,0.35 0.29,0.63 0.64,0.63h4.59c0.43,0 0.8,0.17 1.1,0.46 0.28,0.28 0.44,0.66 0.44,1.09v11.83z"
```

Comment:
Cirrus suggests that we reduce precision as this vector path is very long. Do you think it is reasonable?

Reply:
I looked into this. and did not find a good way of doing this. We have the same warnings on previous icons. However all imported icons were ran through a icon minimizer which made them a bit smaller at least.